routes/auth.js

```diff
@@ -33,6 +33,7 @@ router.post("/register", checkExistingUsers, async (req, res) => {
     try {
         const user = new User();
         user.name = req.body.name;
+        user.slug = req.body.name.toLowerCase().split(" ").join("-"); // lodash
```

Comment:
The package "mongoose-slug-generator" returns a slug automatically. This line is not needed